packages/core/web-dev-server.config.mjs

```diff
@@ -0,0 +1,160 @@
+/**
+ * Base Web Dev Server
+ * 
+ * All dev hosting env in Core uses @web/dev-server including Storybook, Sandbox and Tests
+ */
+
+import { importMapsPlugin } from '@web/dev-server-import-maps';
+import { hmrPlugin, presets } from '@open-wc/dev-server-hmr';
+/** Use Hot Module replacement by adding --hmr to the start command */
+const hmr = process.argv.includes('--hmr');
+
+export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
+  nodeResolve: true,
+  open: './',
+  watch: !hmr,
+  plugins: [
+    importMapsPlugin({
```

Comment:
With web-dev-server all the dev env can share the same config, this includes the unit tests, storybook and the sandbox app.